apps/bfd-server-openapi/pom.xml

```diff
@@ -10,6 +10,10 @@
     <artifactId>bfd-server-openapi</artifactId>
     <packaging>jar</packaging>
 
+    <properties>
+        <sonar.sources>src/main/java,src/main/resources/gov/cms/bfd/server/openapi</sonar.sources>
```

Comment:
This addresses the thymeleaf warning. Setting this as part of the global `sonar.sources` setting causes errors for the directory not existing on the first module encountered in the maven, so we set it here.